### PR TITLE
tutorial for loj1062 added

### PR DESCRIPTION
**Pull request by ssavi-ict**

### CODE REVIEW

**Comment by faiyaz26** [comment]
@Tahanima please review this PR. we will need to fix the images inside the tutorial after it is merged.

File: 1062/en.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+#### Problem Clarification :
```
Comment (Tahanima) [inline 1062/en.md:1]: Please add the problem heading like this - https://github.com/lightoj-dev/problem-tutorials/pull/32/files#diff-94aeb337c51c48839ef777311d73c6d0fb848fcce5914df92d7f1e2973136b9cR1

File: 1062/en.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+<img src="https://latex.codecogs.com/svg.latex?h2&space;=&space;\sqrt{X^{2}}&space;-&space;\sqrt{mid^{2}}" /></a>
+
+So, now you have the value of ```h1``` and ```h2``` . You can easily find ,
+
+<img src="https://latex.codecogs.com/svg.latex?h&space;=&space;\frac{(&space;h1&space;*&space;h2&space;)}{(&space;h1&space;&plus;&space;h2&space;)}" /></a>
+
+For a value of ```mid```, we will found the approximate value of real ```h```. Now for every h, you easily can compare with the original h. 
+
+
```
Comment (Tahanima) [inline 1062/en.md:38]: Please follow this format for Code styling - https://github.com/lightoj-dev/problem-tutorials/pull/32/files#diff-94aeb337c51c48839ef777311d73c6d0fb848fcce5914df92d7f1e2973136b9cR19

Also, please provide the appropriate C++ header files required to get this problem accepted.

**Review by Tahanima — CHANGES_REQUESTED** [review]
Hi
Please resolve the requested changes.

File: 1062/en.md (hunk 1 above)
Comment (ssavi-ict) [reply 1062/en.md:1]: Please Check Now.

File: 1062/en.md (hunk 2 above)
Comment (ssavi-ict) [reply 1062/en.md:40]: Please check Now.

File: 1062/en.md (hunk 1 above)
Comment (faiyaz26) [reply 1062/en.md:1]: I don't see the change @ssavi-ict ? You are supposed to add the problem title like this
```
# Problem Title
```

File: 1062/en.md (hunk 2 above)
Comment (faiyaz26) [reply 1062/en.md:40]: Again, I don't see the changes. You are supposed to add language name after the three ticks. Please check the above link to understand more what we are talking about.

**Review by faiyaz26 — CHANGES_REQUESTED** [review]

**Pull request merged**